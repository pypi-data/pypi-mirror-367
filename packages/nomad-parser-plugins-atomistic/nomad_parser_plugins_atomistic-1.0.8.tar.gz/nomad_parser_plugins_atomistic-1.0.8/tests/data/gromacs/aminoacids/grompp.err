                  :-) GROMACS - gmx grompp, VERSION 5.1.4 (-:

                            GROMACS is written by:
     Emile Apol      Rossen Apostolov  Herman J.C. Berendsen    Par Bjelkmar   
 Aldert van Buuren   Rudi van Drunen     Anton Feenstra   Sebastian Fritsch 
  Gerrit Groenhof   Christoph Junghans   Anca Hamuraru    Vincent Hindriksen
 Dimitrios Karkoulis    Peter Kasson        Jiri Kraus      Carsten Kutzner  
    Per Larsson      Justin A. Lemkul   Magnus Lundborg   Pieter Meulenhoff 
   Erik Marklund      Teemu Murtola       Szilard Pall       Sander Pronk   
   Roland Schulz     Alexey Shvetsov     Michael Shirts     Alfons Sijbers  
   Peter Tieleman    Teemu Virolainen  Christian Wennberg    Maarten Wolf   
                           and the project leaders:
        Mark Abraham, Berk Hess, Erik Lindahl, and David van der Spoel

Copyright (c) 1991-2000, University of Groningen, The Netherlands.
Copyright (c) 2001-2015, The GROMACS development team at
Uppsala University, Stockholm University and
the Royal Institute of Technology, Sweden.
check out http://www.gromacs.org for more information.

GROMACS is free software; you can redistribute it and/or modify it
under the terms of the GNU Lesser General Public License
as published by the Free Software Foundation; either version 2.1
of the License, or (at your option) any later version.

GROMACS:      gmx grompp, VERSION 5.1.4
Executable:   /Users/bonat/GROMACS/BUILD/bin/gmx_mpi
Data prefix:  /Users/bonat/GROMACS/BUILD
Command line:
  gmx_mpi grompp -f ./grompp.mdp -c ./conf -p ./topol -maxwarn 10

Ignoring obsolete mdp entry 'optimize_fft'
Replacing old mdp entry 'verlet-buffer-drift' by 'verlet-buffer-tolerance'
Replacing old mdp entry 'nstxtcout' by 'nstxout-compressed'
Replacing old mdp entry 'xtc-precision' by 'compressed-x-precision'

NOTE 1 [file ./grompp.mdp]:
  The group cutoff scheme is deprecated since GROMACS 5.0 and will be
  removed in a future release when all interaction forms are supported for
  the verlet scheme. The verlet scheme already scales better, and it is
  compatible with GPUs and other accelerators.

Generated 20503 of the 20503 non-bonded parameter combinations
Generating 1-4 interactions: fudge = 1
Generated 17396 of the 20503 1-4 parameter combinations
Excluding 3 bonded neighbours molecule type 'Protein'
Number of degrees of freedom in T-Coupling group Protein is 2531.00

NOTE 2 [file ./grompp.mdp]:
  You are using a plain Coulomb cut-off, which might produce artifacts.
  You might want to consider using PME electrostatics.



There were 2 notes

Thanx for Using GROMACS - Have a Nice Day

