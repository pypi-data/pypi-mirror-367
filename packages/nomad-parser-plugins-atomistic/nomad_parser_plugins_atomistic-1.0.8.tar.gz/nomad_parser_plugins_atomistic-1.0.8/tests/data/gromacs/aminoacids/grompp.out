turning H bonds into constraints...
Analysing residue names:
There are:    64    Protein residues
Analysing Protein...
This run will generate roughly 0 Mb of data
